### ISSUE TITLE
ng e2e with supportFile takes an extra 4 minutes +

### ISSUE THREAD

**Issue by intellix** [nrwl/nx#1976]
_[Please make sure you have read the submission guidelines before posting an issue](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md#-submitting-issue)_

# Prerequisites

Please answer the following questions for yourself before submitting an issue.
**YOU MAY DELETE THE PREREQUISITES SECTION.**

- [x] I am running the latest version
- [ ] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed
- [ ] I'm reporting the issue to the correct repository (not related to Angular, AngularCLI or any dependency)

## Expected Behavior

I should be able to specify a supportFile inside cypress.json to do things like:
```
Cypress.Commands.overwrite('visit', (originalFn, url, options) => {
  return originalFn(url, {
    ...options,
    headers: { 'cache-control': 'no-cache' },
  });
});
```

## Current Behavior

It adds an extra 4 minutes before it starts to run. If you add debug env vars then you can see it going to random non-existent pages for a while.

```
- "supportFile": false,
+ "supportFile": "./src/support/index.ts",
```

**Before:** ng e2e  20.74s user 2.96s system 109% cpu 21.594 total
**After:** ng e2e  22.34s user 3.34s system 9% cpu 4:34.19 total

Even if the supportFile is an empty index.ts or index.js, it still adds an extra 4 minutes before the tests are even run. The video recorded includes 4 minutes of silence.

Basic repository where I generated a project and changed supportFile: https://github.com/intellix/ng-e2e-support-file-bug

**Comment by bcabanes** [comment]
Hi @intellix 
Yes it does in did, we are currently looking into this.

Related issue: https://github.com/nrwl/nx/issues/1976

**Comment by intellix** [comment]
Wrong related issue? points back to here

**Comment by bcabanes** [comment]
Sorry, updated!

**Comment by wvanderdeijl** [comment]
@intellix you could check if the workaround from #2063 works as it feels like the same issue

**Comment by FrozenPandaz** [comment]
Fixed in #1920
